bsi_zoo/tests/test_run_spatial_cv.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+from scipy import linalg
+import pytest
+
+from bsi_zoo.config import get_leadfield_path, get_fwd_fname
+from bsi_zoo.data_generator import get_data
+
+from bsi_zoo.estimators import (
+    run_spatial_cv,
+    iterative_L1,
+    iterative_L2,
+    iterative_sqrt,
+    iterative_L1_typeII,
+    iterative_L2_typeII,
+    gamma_map,
+)
+
+
+@pytest.mark.parametrize("n_times", [5])
+@pytest.mark.parametrize("orientation_type", ["fixed", "free"])
+@pytest.mark.parametrize("nnz", [3])
+# @pytest.mark.parametrize("subject", [None, "CC120166"])
+@pytest.mark.parametrize("subject", [None])
+@pytest.mark.parametrize(
+    "solver,alpha,rtol,atol,cov_type,extra_params",
+    [
+        (iterative_L1, 0.1, 1e-1, 5e-1, "diag", {}),
+        (iterative_L2, 0.1, 1e-1, 5e-1, "diag", {}),
+        (iterative_sqrt, 0.1, 1e-1, 5e-1, "diag", {}),
+        (iterative_L1_typeII, 0.1, 1e-1, 5e-1, "full", {}),
+        (iterative_L2_typeII, 0.1, 1e-1, 5e-1, "full", {}),
+        (gamma_map, 0.2, 1e-1, 5e-1, "full", {"update_mode": 1}),
+        (gamma_map, 0.2, 1e-1, 5e-1, "full", {"update_mode": 2}),
+        (gamma_map, 0.2, 1e-1, 5e-1, "full", {"update_mode": 3}),
+    ],
+)
+def test_run_spatial_cv(
+    n_times,
+    solver,
+    alpha,
+    rtol,
+    atol,
+    cov_type,
+    subject,
+    nnz,
+    orientation_type,
+    extra_params,
+    save_estimates=False,
+):
+
+    path_to_leadfield = get_leadfield_path(
+        subject, orientation_type
+    )  # setting leadfield paths
+
+    n_orient = 1 if orientation_type == "fixed" else 3
```

Comment:
This test should not necessitate to use stc objects. I would just check that coef_ is the solver is ok in some sense